article/Introduction/pre-speciale-report.tex

```diff
@@ -0,0 +1,12 @@
+\subsection{Pre specialization paper}
+In preparation for our master thesis we spend a semester gaining knowledge in the field of recommendation systems.
+We wrote a paper called \textit{Exploring Recommender Systems with Graph Convolutional Networks}.
+In this paper we studied recommendation systems using GCN's to achieve state of the art results.
+In particular we looked at the models NGCF, PUP and LightGCN.
+PUP and LightGCN is both inspired by NGCF, where LightGCN removes complexity operations from NGCF and PUP adds side information in form of prices and categories.
+Both achieved better performance in NDCG compared to NGCF.
+Because of this we thought it would be interesting to extend LightGCN with price and categories to see if it would improve performance.
```

Comment:
```suggestion
Because of this, we thought it would be interesting to extend LightGCN with price and categories to see if it would improve performance.
```